packages/arb-token-bridge-ui/src/components/TransferPanel/TokenButton.tsx

```diff
@@ -73,16 +73,17 @@ export function TokenButton(): JSX.Element {
               aria-label="Select Token"
             >
               <div className="flex items-center gap-2">
-                {tokenLogo && (
-                  // SafeImage is used for token logo, we don't know at buildtime where those images will be loaded from
-                  // It would throw error if it's loaded from external domains
-                  // eslint-disable-next-line @next/next/no-img-element
-                  <img
+                {/* {tokenLogo && ( 
+                 // SafeImage is used for token logo, we don't know at buildtime
+                where those images will be loaded from // It would throw error
+                if it's loaded from external domains // eslint-disable-next-line
+                @next/next/no-img-element 
+                 <img
                     src={tokenLogo}
                     alt="Token logo"
                     className="h-5 w-5 sm:h-7 sm:w-7"
                   />
-                )}
+                )} */}
```

Comment:
commenting it out for now until we update the token image source files to be of better quality

Reply:
can we add a comment for this in the code